### PR TITLE
Shield Reworks

### PR DESCRIPTION
**Pull request by ProphetCrow**
## About the PR
Reopens (https://github.com/echo-station/echo-station/pull/37)

## Why / Balance
Shields are very brittle and will shatter way before the user goes into a critical state, regardless if the user was using the supposedly correct shield for the correct damage type. This PR ultimately seeks to make shields a more useful tool with an ultimately longer shelf life, with respect to their niche specialization.

Shields now are able to be stored in backpacks, consuming a 5x3 grid (taller than wide). This idea is ultimately aimed towards user ergonomics and quality of life, because without this change, you would need to sacrifice a hand slot to bring a shield with you to any environment.
Further justification: The Energy shield and the telescopic shield are able to be stored in a backpack while deployed, and they consume a 4x4 grid, which is also the same size as a blood-red hardsuit and most large weapons.
This change is also aimed towards a future weapon size rebalance which will aim to convert most rifle sized weapons to 5x2 grid slots.

**Changelog**
:cl:
- tweak: Shield stats have been modified

### CODE REVIEW

**Comment by redmushie** [comment]
Please fix the failing checks, then I can merge it. Thanks!

**Comment by redmushie** [comment]
Please add Echo markers, preferably at the first line of your changes if possible, so I know what the reason for the change is / what to change when I merge upstream changes

**Review by redmushie — CHANGES_REQUESTED** [review]
Please add Echo markers, preferably at the first line of your changes if possible, so I know what the reason for the change is / what to change when I merge upstream changes

**Comment by ProphetCrow** [comment]
see 33d22aa

File: Resources/Prototypes/Entities/Objects/Shields/shields.yml [hunk 1]
```diff
@@ -249,32 +225,26 @@
...
-          Slash: 0.85
-          Piercing: 0.85
+          Blunt: 0.7 #Echo
+          Slash: 0.7
+          Piercing: 0.7
         flatReductions:
-          Blunt: 0.5
-          Slash: 0.5
-          Piercing: 0.5
+          Blunt: 1
```
Comment (redmushie) [inline Resources/Prototypes/Entities/Objects/Shields/shields.yml]: Marker

File: Resources/Prototypes/Entities/Objects/Shields/shields.yml [hunk 2]
```diff
@@ -96,18 +81,18 @@
...
         coefficients:
-          Heat: 0.7
+          Heat: 0.3 #Echo - only good at stopping lasers / heat attacks
       activeBlockModifier:
         coefficients:
-          Heat: 0.5
+          Heat: 0.3 #Echo
         flatReductions:
-          Heat: 1
+          Heat: 2
```
Comment (redmushie) [inline Resources/Prototypes/Entities/Objects/Shields/shields.yml]: Marker

File: Resources/Prototypes/Entities/Objects/Shields/shields.yml [hunk 3]
```diff
@@ -116,12 +101,15 @@
...
+          Blunt: 0.7 #Echo Good against bullets, blunt DR here for future proofing
+          Piercing: 0.4
       activeBlockModifier:
         coefficients:
-          Piercing: 0.5
+          Blunt: 0.7 #Echo
+          Piercing: 0.4
         flatReductions:
-          Piercing: 1
+          Blunt: 1.5
```
Comment (redmushie) [inline Resources/Prototypes/Entities/Objects/Shields/shields.yml]: Marker

**Review by redmushie — CHANGES_REQUESTED** [review]

**Review by redmushie — COMMENTED** [review]
LGTM!

**Review by redmushie — APPROVED** [review]

**Pull request merged**